engine/class_modules/sc_mage.cpp

```diff
@@ -3498,10 +3493,13 @@ struct arcane_barrage_t final : public dematerialize_spell_t
     {
       p()->trigger_clearcasting( 1.0, 0_ms );
       p()->trigger_arcane_charge( arcane_soul_charges );
+      p()->buffs.arcane_soul_counter->trigger( p()->buffs.arcane_soul->remains() );
     }
 
     p()->trigger_spellfire_spheres();
     p()->trigger_mana_cascade();
+    p()->consume_burden_of_power();
+    consume_nether_precision( target, true );
```

Comment:
Calling `consume_nether_precision` with `true` as the 2nd argument doesn't make sense here. With AV gone, the second param should just be removed.